### DISCUSSION TITLE
Scanner direction arrow changes into objective arrow

### DISCUSSION THREAD

**Discussion by mefistotelis** [swfans/syndwarsfx#206] [General]
You see that white arrow showing base direction on scanner map? Type "3.1415926535" using numpad.

It doesn't seem to work on all objectives - sometimes the arrow just disappears. But the idea is it will now point to the nearest target.

**Comment by Moburma** [comment]
Wow, I've said it before, but it feels like there's an almost infinite number of bizarre hidden features in this game. In a hundred years they will have solved every riddle of ancient Egypt, but still be finding weird stuff in Syndicate Wars.

Interesting it's the values of Pi as well, of course there is that file on the CD with the largest prime at the time, someone on the team was clearly a maths nerd.

**Comment by mefistotelis** [comment]
The pi is used often around various calculations in games, so not sure if it even required someone with a math knack. I found it in the scanner code, which draws a lot of circles and arches -> pi. The scanner map rotates with the view -> pi. The triangular arrow shows direction between two points -> pi. There's pi in every other function there.

(though Bullfrog used constant point math, where pi equals 1024 rather than its normal value)
